interface/usergroup/usergroup_admin_add.php

```diff
@@ -158,17 +154,16 @@ function authorized_clicked() {
 <table border=0>
 
 <tr><td valign=top>
-<form name='new_user' method='post'  target="_parent" action="usergroup_admin.php"
+<form name='new_user' id="new_user" method='post'  target="_parent" action="usergroup_admin.php"
  onsubmit='return top.restoreSession()'>
 <input type='hidden' name='mode' value='new_user'>
 <input type='hidden' name='secure_pwd' value="<?php echo $GLOBALS['secure_password']; ?>">
 
 <span class="bold">&nbsp;</span>
-</td><td>
 <table border=0 cellpadding=0 cellspacing=0 style="width:600px;">
 <tr>
-<td style="width:150px;"><span class="text"><?php xl('Username','e'); ?>: </span></td><td  style="width:220px;"><input type=entry name=rumple style="width:120px;"> <span class="mandatory">&nbsp;*</span></td>
-<td style="width:150px;"><span class="text"><?php xl('Password','e'); ?>: </span></td><td style="width:250px;"><input type="entry" style="width:120px;" name=stiltskin><span class="mandatory">&nbsp;*</span></td>
+<td style="width:150px;"><span class="text"><?php xl('Username','e'); ?>: </span></td><td  style="width:220px;"><input type=entry name="rumple" style="width:120px;"> <span class="mandatory">&nbsp;*</span></td>
+<td style="width:150px;"><span class="text"><?php xl('Password','e'); ?>: </span></td><td style="width:250px;"><input type=entry style="width:120px;" name="stiltskin"><span class="mandatory">&nbsp;*</span></td>
 </tr>
```

Comment:
For both the add user and edit user screens:
If the username, f name, or l name are not populated then only a red box shows up (there is no associated error text).